### DISCUSSION TITLE
Is the creation of Sub Accounts possible?

### DISCUSSION THREAD

**Discussion by ErminMerdanovic** [drakkan/sftpgo#1402] [Q&A]
Hello everyone, I'm relatively new to using sftpgo, and I'm curious if it's possible for a user with an account to create sub-accounts. I'm thinking of a feature similar to the sharing function, but accessible through sftp instead of a the URL/web interface. Your insights are much appreciated!

**Comment by EnochPrime** [comment]
I am new to SFTPGo as well, but these appears to be possible with the [admin roles](https://github.com/drakkan/sftpgo/blob/main/docs/roles.md).

What you would need to do is login to the admin panel and create a new admin account for the user that you want to allow to make "subusers". Then you need to make a new role and assign that to the new admin. Then new admin can login to the admin panel and create any users they wish. These new users will automatically be assigned the same role so can be managed by this new admin.

**Comment by ErminMerdanovic** [comment]
@EnochPrime Yeah, but then I wouldn't be able to set an overall quota for that admin user.
